pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -448,6 +451,16 @@ type JaegerCassandraCreateSchemaSpec struct {
 	TTLSecondsAfterFinished *int32 `json:"ttlSecondsAfterFinished,omitempty"`
 }
 
+// GRPCPluginSpec represents the GRPC Plugin init container
+// +k8s:openapi-gen=true
+type GRPCPluginSpec struct {
+	Image string `json:"image,omitempty"`
+
+	ConfigurationFile string `json:"configurationFile,omitempty"`
```

Comment:
I think I initially used options (because it was the least effort) and @pavolloffay suggested they should be CR properties beause they are required.

Do you remember @pavolloffay ? :)

Reply:
If this has been discussed before, keep what you agreed upon then.

Reply:
Actually I can't find anywhere Pavol said that. If you prefer flags I can change to that.

Reply:
It's a comment to the PR: https://github.com/jaegertracing/jaeger-operator/pull/1045#issuecomment-627177150